City-of-Helsinki/berth-reservations-admin
# VEN-621 | Order info on the offer card

## Description :sparkles:
- Fetch order info on the offer card
- Implement editing modal for creating and deleting `orderLine`s

### Closes :no_good_woman:
**[VEN-621](https://helsinkisolutionoffice.atlassian.net/browse/VEN-621)**: FE - Offer card: replace hardcoded invoice data with GQL 

### Related :handshake:

## Testing :alembic:

### Automated tests :gear:️

### Manual testing :construction_worker_man:
- Go to a single application page
- `TARJOUS` card should show the order info
- Click on `Muokkaa` to open an editing modal

## Screenshots :camera_flash:
![image](https://user-images.githubusercontent.com/23040926/86223570-673da780-bb90-11ea-8641-cf83a679457b.png)
![image](https://user-images.githubusercontent.com/23040926/86223624-74f32d00-bb90-11ea-944f-59489d361612.png)

## Additional notes :spiral_notepad:
